packages/fetch/src/response.js

```diff
@@ -126,6 +126,23 @@ export default class Response extends Body {
 		});
 	}
 
+	/**
+	 * @param {any} data The URL that the new response is to originate from.
+	 * @param {ResponseInit} [responseInit] An optional status code for the response (e.g., 302.)
+	 * @returns {Response} A Response object.
+	 */
+	static json(data, responseInit = {}) {
```

Comment:
I *think* this should be generic but I'm not positive. We should probably make sure this works with our new loader type inference.

Reply:
ha, totally forgot about this PR lol, I'll pull in the generic additions from `json` in the remix repo shortly